requirements.txt

```diff
@@ -22,6 +22,7 @@ pyclipper==1.1.0.post3    # via booleanoperations
 pyparsing==2.4.7          # via vttlib
 pytz==2020.1              # via fs
 six==1.15.0               # via fs
+skia-pathops==0.4.1     
```

Comment:
Add this in requirements.in without the version pin and do `pip-compile requirements.in`.

Reply:
Updated